package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -474,6 +493,10 @@ def _reopen(self):
         self.ts = self._Timestep(self.n_atoms, **self._ts_kwargs)
         self.ts.frame = -1
 
+    def _get_column(self, ids, col_ids):
```

Comment:
This is just a convenience function, I'd just inline it with a lambda where it's used